pysnooper/tracer.py

```
@@ -7,14 +7,15 @@
 import collections
 import datetime as datetime_module
 
-def get_shortish_repr(item) -> str:
+import six
+
+def get_shortish_repr(item):
     r = repr(item)
     if len(r) > 100:
         r = '{r[:97]}...'.format(**locals())
```

Ruff (0.12.2)

`.format` call has invalid format string: Expected '}' before end of string

(F521)